app/views/ParentsView.coffee

```diff
@@ -0,0 +1,17 @@
+RootComponent = require 'views/core/RootComponent'
+template = require 'templates/base-flat'
+ParentsViewComponent = require('./ParentsViewComponent.vue').default
+ParentReferTeacherModal = require('views/core/ParentReferTeacherModal')
+
+module.exports = class ParentView extends RootComponent
+  id: 'parents-view'
+  template: template
+  VueComponent: ParentsViewComponent
+    
+  afterRender: ->
+    super()
+    if (@vueComponent)
+      @vueComponent.$on('referTeacher', => @onReferTeacher())
```

Comment:
Do we want to call this more than once?  `afterRender()` can usually be called multiple times.